feat: suppress FFMPEG output

Om1609 opened the pull request:
## Summary

Suppress FFMPEG output when audio is finished recording
<!-- What is this pull request for? Does it fix any issues? -->

## Information

<!-- Put an x inside [ ] to check it, like so: [x] -->

- [x] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed).
- [ ] This PR is **not** a code change (e.g. documentation, README, typehinting,
      examples, ...).

## Checklist

<!-- Put an x inside [ ] to check it, like so: [x] -->

- [ ] I have searched the open pull requests for duplicates.
- [x] If code changes were made then they have been tested.
  - [ ] I have updated the documentation to reflect the changes.
- [ ] If `type: ignore` comments were used, a comment is also left explaining why.
- [ ] I have updated the changelog to include these changes.

plun1331 reviewed (dismissed):
Please add a changelog

Om1609 commented:
> you updated the call args, so update

Uh. Just found it is Added, Removed, Fixed and Changed from previous changelogs. 

Have put in Changed for the time being
